File: test/end-to-end/Lists/C#/Album.cs
namespace Lists
{
    public class Album
    {
        public string Name;
        public int Year;

        public Album(string name, int year)
        {
            this.Name = name;
            this.Year = year;
        }

        public string GetLabel()
        {
            return string.Format("{0}: {1}", this.Name, this.Year);
        }
    }
}

File: test/end-to-end/Lists/Java/Album.java
package lists;

public class Album {
    public String name;
    public Integer year;

    public Album(String name, Integer year) {
        this.name = name;
        this.year = year;
    }

    public String getLabel() {
        return String.format("$s: $d", this.name, this.year);
    }
}
